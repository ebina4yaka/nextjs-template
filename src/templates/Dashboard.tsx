import { ReactElement } from 'react'
import Copyright from '../atoms/Copyright'
import Navbar from '../organisms/Navbar'

export default function Dashboard(): ReactElement {
  return (
    <>
      <header>
        <Navbar />
      </header>
      <main>
        <div className="lg:container mx-auto">
          <div className="md:box-content">
            <div className="font-sans text-4xl text-center m-10">
              This is Title
            </div>
          </div>
        </div>
      </main>
      <footer className="text-center">
        <Copyright />
      </footer>
    </>
  )
}
