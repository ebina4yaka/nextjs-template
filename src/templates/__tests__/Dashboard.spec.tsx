import renderer from 'react-test-renderer'
import Dashboard from '../Dashboard'

jest.mock('../../organisms/Navbar', () => 'MockNavbar')
jest.mock('../../atoms/Copyright', () => 'MockCopyright')

it('Snapshot test for Dashboard', () => {
  const tree = renderer.create(<Dashboard />).toJSON()
  expect(tree).toMatchSnapshot()
})
