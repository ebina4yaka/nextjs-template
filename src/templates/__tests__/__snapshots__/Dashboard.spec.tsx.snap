// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Dashboard 1`] = `
Array [
  <header>
    <MockNavbar />
  </header>,
  <main>
    <div
      className="lg:container mx-auto"
    >
      <div
        className="md:box-content"
      >
        <div
          className="font-sans text-4xl text-center m-10"
        >
          This is Title
        </div>
      </div>
    </div>
  </main>,
  <footer
    className="text-center"
  >
    <MockCopyright />
  </footer>,
]
`;
